sample_consensus/include/pcl/sample_consensus/impl/mlesac.hpp

```diff
@@ -66,6 +66,7 @@ pcl::MaximumLikelihoodSampleConsensus<PointT>::computeModel (int debug_verbosity
 
   // Compute sigma - remember to set threshold_ correctly !
   sigma_ = computeMedianAbsoluteDeviation (sac_model_->getInputCloud (), sac_model_->getIndices (), threshold_);
+  const double prob_factor1 = -1.0 / (2.0 * sigma_ * sigma_);
```

Comment:
`distance_scaling_parameter` or similar? Since it's applied to the distance from mean in a normal distribution

Reply:
Renamed to `dist_scaling_factor`. IMO a parameter is something the user sets.